BL_table/BL_table.md

```diff
@@ -0,0 +1,283 @@
+Make a baseline table
+================
+Author
+13.5.2022
+
+# Background
+
+Epidemiologic and clinical research papers often describe the study
+sample in a first, baseline table. Providing a baseline table is one of
+the recommendations of the CONSORT (Moher et al. (2012)) and STROBE (von
+Elm et al. (2007)) statements. If well-executed, it provides a rapid,
+objective, and coherent grasp of the data and can illuminate potential
+threats to internal and external validity (Hayes-Larson et al. (2019)).
+
+# Main important considerations
+
+Baseline tables can be presented in different ways, but certain features
+are common and some recommendations need be followed. In the next
+paragraph and figure we provide an overview of the main important
+considerations, for more detailed review and guidelines see (e.g. Moher
+et al. (2012), Hayes-Larson et al. (2019)).
+
+Characteristics are generally presented on the left as rows and
+groupings at the top as columns. Groups are defined by treatment
+allocation for RCTs, disease status for case–control studies or
+cohorts.If appropriated, the overall population’ characteristics may be
+presented in a separate column. Characteristics should include of study
+participants (e.g., demographic, clinical, social) and information on
+exposures and potential data confounders. Inside each cell, descriptive
+statistics are typically given as n (%) for categorical variables and
+mean (standard deviation) or median (25th-75th percentile or
+minimum-maximum) for continuous variables.The type of statistical
+measurement used should be indicated (e.g., together with the
+characteristic, in a footnote).The number of participants with missing
+data should be reported for each variable of interest.
+
+**Important note: in RCT’s, differences between randomized groups should
+not be tested for statistical significance**. Indeed, some imbalance is
+unavoidable owing to chance, especially if many characteristics are
+considered (Palesch (2014))
```

Comment:
Also STROBE advises against tests in baseline tables, see under 14 (a) of https://journals.lww.com/epidem/Fulltext/2007/11000/Strengthening_the_Reporting_of_Observational.28.aspx :
"Inferential measures such as standard errors and confidence intervals should not be used to describe the variability of characteristics, and significance tests should be avoided in descriptive tables. "

Reply:
I agree with Andrea, I always point to STROBE, item 14 a (maybe 14 b and c too), to convince people that they do not need / should not present p-values in descriptive tables on observational study characteristics...

Reply:
OK I tried to update the text tell me if it fine like this.